subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -488,13 +1076,45 @@ static int fp_storage_ak_reset(void)
 		}
 	}
 
+	if (IS_ENABLED(CONFIG_BT_FAST_PAIR_STORAGE_AK_BOND)) {
+		for (uint8_t index = 0; index < ARRAY_SIZE(fp_bonds); index++) {
```

Comment:
you consider defining an iterator instead of a standard for loop to make the code more modular